src/components/MapsSection/MapsSection.styles.tsx

```diff
@@ -77,7 +77,17 @@ export const TagButton = styled.span<{ active?: string }>`
   box-sizing: border-box;
   background-color: #ffffff95;
 
-  cursor: ${({ active }) => active !== "true" && "pointer"};
+  cursor: ${({ active, isLoading }) => {
```

Comment:
i think we can change this part, just to follow the pattern 
`cursor: ${({ active, isLoading }) => 
  isLoading ? "not-allowed" : active === "true" ? "" : "pointer"
};`

Reply:
done.